estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,112 @@
+                <field name="expected_price" />
+                <field name="bedrooms" />
+                <field name="living_area" />
+                <field name="facades" />
+                <field name="property_type" />
+
+
+                <filter string="Status" name="state"
+                    domain="[('state','in',['new','offer_received'])]" />
+                <group string="Group By">
+                    <filter string="Postcode" name="group_by_postcode"
+                        context="{'group_by':'postcode', 'residual_visible':True}" />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="estate_property_form" model="ir.ui.view">
+        <field name="name">estate.property.form</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="edit_form">
+                <header>
+                    <button name="action_cancel" type="object" string="Cancel" />
+                    <button name="action_sold" type="object" string="Sold" />
+                </header>
+                <sheet>
+                    <group>
+                        <h1>
+                            <field name="name"></field>
+                        </h1>
+
+                        <field name="tag_ids" widget="many2many_tags" />
+                    </group>
+                    <group>
+                        <group>
+                            <field name="property_type" />
+                            <field name="postcode" />
+                            <field name="date_availaility" string="Available From" />
+                        </group>
+                        <group>
+                            <field name="expected_price" />
+                            <field name="best_price" />
+                            <field name="selling_price" />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Description">
+                            <group>
+                                <field name="description" />
+                                <field name="bedrooms" />
+                                <field name="living_area" string="Living Area(sqm)" />
+                                <field name="facades" />
+                                <field name="garage" />
+                                <field name="garden" />
+                                <field name="garden_area" string="Garden Area (sqm)" />
+                                <field name="garden_orientation" string="Garden Orientation" />
+                                <field name="active" />
+                                <field name="state" />
+                                <field name="total_area" />
+                            </group>
+                        </page>
+                        <page string="Offers">
+                            <field name="offers_ids" />
```

Comment:
Should be able to add an offer directly from the list instead of opening the wizard.